### PR TITLE
Adding postgresql instructions

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,42 @@ Content Sources is an application for storing information about external content
 
 ##Developing
 
-Create a `.env` file with:
-~~~
+### Configuring Postgresql
+
+As root run: 
+```shell
+yum install -y postgreql
```
Comment (Andrewgdewar) [inline README.md:13]: postgresql?

File: README.md [hunk 2]
```diff
@@ -6,18 +6,42 @@ Content Sources is an application for storing information about external content
...
+
+echo "host    all             all             127.0.0.1/32            trust" >> /var/lib/pgsql/data/pg_hba.conf
+echo "host    all             all             ::1/128                 trust" >> /var/lib/pgsql/data/pg_hba.conf
+systemctl start postgresql
+systemctl enable postgreql
```
Comment (Andrewgdewar) [inline README.md:18]: postgresql

File: README.md [hunk 3]
```diff
@@ -6,18 +6,44 @@ Content Sources is an application for storing information about external content
...
+```shell
+yum install -y postgresql
+
+echo "host    all             all             127.0.0.1/32            trust" >> /var/lib/pgsql/data/pg_hba.conf
+echo "host    all             all             ::1/128                 trust" >> /var/lib/pgsql/data/pg_hba.conf
+systemctl start postgresql
+systemctl enable postgresql
+sudo -u postgres createdb content
+sudo -u postgres psql -c "CREATE USER content2 WITH PASSWORD 'content'"
+sudo -u postgres psql -c "grant all privileges on database content TO content"
```
Comment (Andrewgdewar) [inline README.md:21]: grant all privileges on database content TO content2

Comment (jlsherrill) [reply README.md:21]: fixed :)

**Comment by jlsherrill** [comment]
ended up putting all this in https://github.com/content-services/content-sources-backend/pull/20

**Pull request closed without merging**